napari/_qt/qt_viewer.py

```diff
@@ -50,22 +55,13 @@ def __init__(self, viewer):
         self.canvas.connect(self.on_draw)
 
         self.view = self.canvas.central_widget.add_view()
-
-        # Set 2D camera (the camera will scale to the contents in the scene)
-        self.view.camera = PanZoomCamera(aspect=1)
-        # flip y-axis to have correct aligment
-        self.view.camera.flip = (0, 1, 0)
-        self.view.camera.set_range()
-        self.view.camera.viewbox_key_event = viewbox_key_event
-
-        # TO DO: Remove
-        self.viewer._view = self.view
+        self.set_camera()
+        viewer.camera = self.view.camera
```

Comment:
I'd prefer to avoid adding the `camera` attribute directly to the viewer. We're going to be refactoring to remove the `view` from the `Viewer` too, but this should not be needed. If you need something from the `Viewer` to communicate with the `camera` we should be using events

Reply:
removed